guides/gatherers.md

```diff
@@ -14,32 +14,49 @@ Facts Gathering process in a nutshell
 fact = gatherer(argument)
 ```
 
+## A note of gatherers versioning
+
+When a gatherer includes a breaking change (e.g. the Rhai output format changes), a new version of that same gatherer will be introduced with a new
+version labeled as a suffix to the gatherers name.
+
+Example:
+
+- `systemd@v1` -> Represents the first version of the systemd gatherer
+- `systemd@v2` -> Represents the second version of the systemd gatherer
+
+Note that when writing a check, if no tag is specified (e.g. `systemd`), the latest version will be used. Because of this it is **strongly** recommended
```

Comment:
What about skipping  because ? 
Before: `Because of this it is strongly recommended ...`
After: `It is strongly recommended`